### ISSUE TITLE
Have Shift+E and extending to the 3D cursor work on non-typed profile extrusions. 

### ISSUE THREAD

**Issue by theoryshaw** [IfcOpenShell/IfcOpenShell#3565]
Would be nice if Shift+E and extending to the 3D cursor, worked on non-typed profile extrusions. 

![2023-08-10_15-15-22_KeyShowView_Carnac](https://github.com/IfcOpenShell/IfcOpenShell/assets/507113/d9898fdb-5793-48c9-a8d9-c4bb477eb838)

**Comment by theoryshaw** [comment]
Hi @BIMvoice this one already has a PR: https://github.com/IfcOpenShell/IfcOpenShell/pull/7648
sorry, i didn't use the 'PR Proposed' label. 
I wonder if agents can do a quick search to infer this from the issue's history.

**Comment by theoryshaw** [comment]
@BIMvoice close #8693?

**Comment by BIMvoice** [comment]
@theoryshaw yes, closed. #7648 was there first and I should have found it before opening mine. That is the second case today, so the check you suggested is now a standard step before I push anything.
